static/css/restyle/restyle.less

```diff
@@ -1994,6 +1999,10 @@ body.editor-tools {
         border-radius: 0;
       }
 
+      .results .files {
+        width: 18em;
```

Comment:
Better still using pixels would probably be less ambiguous for a width unless you really need it to be font-size dependent.